weldx/transformations/local_cs.py

```diff
@@ -799,9 +907,23 @@ def interp_time(
             time = time + time_ref
 
         orientation = ut.xr_interp_orientation_in_time(self.orientation, time)
-        coordinates = ut.xr_interp_coordinates_in_time(self.coordinates, time)
+        if isinstance(self.coordinates, TimeSeries):
+            time_interp = time
+            if isinstance(time_interp, pd.DatetimeIndex):
+                time_interp = time - self.reference_time
 
-        return LocalCoordinateSystem(orientation, coordinates, time_ref=time_ref)
+            coordinates = self._coords_from_discrete_time_series(
+                self.coordinates.interp_time(time_interp)
+            )
+
+            if self.has_reference_time:
+                coordinates.weldx.time_ref = self.reference_time
```

Comment:
The condition wasn't necessary before using the weldx accessor. The problem is, that it doesn't support `None`. There is a to-do in the implementation of the accessor addressing this:

~~~ python
@time_ref.setter
    def time_ref(self, value: pd.Timestamp):
        """Convert INPLACE to new reference time.

        If no reference time exists, the new value will be assigned
        TODO: should None be allowed and pass through or raise TypeError ?
        """
        if "time" in self._obj.coords:
            value = _as_valid_timestamp(value)
            if self._obj.weldx.time_ref and is_timedelta64_dtype(self._obj.time):
                if value == self._obj.weldx.time_ref:
                    return
                _attrs = self._obj.time.attrs
                time_delta = value - self._obj.weldx.time_ref
                self._obj["time"] = self._obj.time.data - time_delta
                self._obj.time.attrs = _attrs  # restore old attributes !
                self._obj.time.attrs["time_ref"] = value  # set new time_ref value
            else:
                self._obj.time.attrs["time_ref"] = value
~~~

Do we want to make a decision and clear this to-do?

Reply:
what was the previous behavior passing `None`?

Reply:
Crash. I believe it was the timestamp conversion that failed.

Reply:
what do you think default behavior should be when passing `None`?

remove an existing reference time (how else would we do that) or just do nothing?

Reply:
Initially, I added 

~~~ python
if value is None:
    return
~~~

but I wasn't sure because this caused a Failure in the exception tests of the `reset_reference_time` method. Currently, we expect it to fail if None is passed. Would be conservative about it and make raising an exception the default behavior.